README.md

```diff
@@ -11,6 +11,7 @@
 | ------------------- | ---------------------------------------------------------------------------------------------------- |
 | Website             | [https://tgstation13.org](https://tgstation13.org)                                                   |
 | Code                | [https://github.com/tgstation/tgstation](https://github.com/tgstation/tgstation)                     |
+| Server Config       | [https://github.com/tgstation-operations/server-config](https://github.com/tgstation-operations)     |
```

Comment:
You're just linking to the operations org itself

Reply:
Should be good now